integreat_cms/core/settings.py

```diff
@@ -151,6 +151,18 @@
     "INTEGREAT_CMS_NOMINATIM_API_URL", "http://nominatim.maps.tuerantuer.org/nominatim/"
 )
 
+###############
+# TEXTLAB API #
+###############
+
+#: URL to the textlab api
+TEXTLAB_API_URL = "https://preview.text-lab.de/api"
+
+#: Password for the textlab api
+TEXTLAB_API_PASSWORD = os.environ.get("TEXTLAB_PW")
+
+#: Username for the textlab api
+TEXTLAB_API_USERNAME = "Integreat"
```

Comment:
```suggestion
TEXTLAB_API_USERNAME = os.environ.get("INTEGREAT_CMS_TEXTLAB_API_USERNAME", "Integreat")
```
I would make this value configurable, in case somebody wants to use the CMS and does not have access to Integreat's credentials